src/domains/checkout-app/api/checkout/checkout_feature_flags/v1/_openapi.json.tpl

```diff
@@ -0,0 +1,141 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+        "version": "1.0.0",
+        "title": "Pagopa Feature Flags",
+        "description": "This API set contains APIM level apis used to manage feature flags for checkout."
+    },
+    "servers": [
+        {
+            "url": "https://${host}"
+        }
+    ],
+    "tags": [
+        {
+            "name": "featureFlags",
+            "description": "This API is used to handle feature flags for checkout, and follows the Openfeature specifications.",
+            "externalDocs": {
+                "url": "https://openfeature.dev",
+                "description": "Openfeature"
+            }
+        }
+    ],
+    "paths": {
+        "/features/{featureKey}/enabled": {
+            "get": {
+                "tags": ["featureFlags"],
+                "summary": "Get feature flag status",
+                "description": "Returns the status of the specified feature flag.",
+                "parameters": [
+                    {
+                        "name": "featureKey",
+                        "in": "path",
+                        "required": true,
+                        "schema": {
+                            "type": "string"
+                        },
+                        "description": "The key of the feature flag to check."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Feature flag status",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/FeatureFlagResponse"
+                                },
+                                "example": {
+                                    "featureKey": "newCheckoutFlow",
```

Comment:
fix this example since it's not conforming to the response (remove featureKey field)